### ISSUE TITLE
Accessibility Insights for Web

### ISSUE THREAD

**Issue by hamza221** [nextcloud/contacts#4494]
The result of manual + automated accessibility testing using Accessibility Insights for Web  (6 failing tests) 

[QuickAssess_20250604_ContactsNextcloud.html.txt](https://github.com/user-attachments/files/20593475/QuickAssess_20250604_ContactsNextcloud.html.txt)
